crates/biome_service/src/configuration/linter/mod.rs

```diff
@@ -197,7 +211,7 @@ impl From<&RulePlainConfiguration> for Severity {
     }
 }
 
-#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[derive(Clone, Copy, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
```

Comment:
Is `Copy` needed?

Reply:
I could use `clone` where we need to. This just simplifies some code path.

Reply:
Basically I don't see any value of not implementing `Copy` for 1 byte-sized types.